### PR TITLE
feat: Adding berachain

### PR DESCRIPTION
**Pull request by dennyscode**
Ticket: https://lifi.atlassian.net/browse/LF-10392

### CODE REVIEW

File: src/components/Berachain/components/BerachainMarkets/BerachainMarketsFilters/BerachainSearch.tsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+      onInputChange={(event, value) => {
+        setIsInputEmpty(value.length === 0);
+        if (search === value) {
+          setSearch(undefined);
+        } else {
+          setSearch(value);
+        }
+      }}
+      slotProps={{
+        paper: {
```
Comment (tcheee) [inline src/components/Berachain/components/BerachainMarkets/BerachainMarketsFilters/BerachainSearch.tsx:42]: can we split it in its own component ?

File: src/components/Berachain/components/BerachainMarketCard/BerachainMarketCard.tsx [hunk 2]
```diff
@@ -234,7 +236,7 @@ export const BerachainMarketCard = ({
               <BerachainProgressCard
                 title={'Net APY'}
                 value={`${typeof apys?.total === 'number' ? apys?.total + '%' : apys?.total || 'N/A'}`}
-                tooltip={'Net APY tooltip msg lorem ipsum'}
+                tooltip={'Expected return rate on your invested amount.'}
```
Comment (dennyscode) [inline src/components/Berachain/components/BerachainMarketCard/BerachainMarketCard.tsx:239]: Idea: 
`Annual yield after fees`

File: src/components/Berachain/components/BerachainMarkets/BerachainMarketsHeader.tsx [hunk 3]
```diff
@@ -47,13 +47,13 @@ export const BerachainMarketsHeader = () => {
         <BerachainProgressCard
           title={'Total Value Locked'}
           value={'$107.07K'}
-          tooltip={'Lorem ipsum..'}
+          tooltip={'This is the total amount in a protocol.'}
```
Comment (dennyscode) [inline src/components/Berachain/components/BerachainMarkets/BerachainMarketsHeader.tsx:50]: Idea:
`Assets staked or locked`

File: src/components/Berachain/components/BerachainMarkets/BerachainMarketsHeader.tsx [hunk 4]
```diff
@@ -47,13 +47,13 @@ export const BerachainMarketsHeader = () => {
...
           sx={BerachainMarketHeaderProgressCardStyles(theme)}
         />
         <BerachainProgressCard
           title={'Total Volume'}
           value={'$406.82K'}
-          tooltip={'Lorem ipsum..'}
+          tooltip={'This is the total volume done on Boyco.'}
```
Comment (dennyscode) [inline src/components/Berachain/components/BerachainMarkets/BerachainMarketsHeader.tsx:56]: Idea:
`Total value of transactions processed on the platform`

**Pull request closed without merging**